Cargo.toml

```diff
@@ -49,110 +49,112 @@ funty = { git = "https://github.com/bitvecto-rs/funty/", rev = "7ef0d890fbcd8b3d
-git-version           = "0.3"
-hex                   = "0.4"
-http                  = "0.2.8"
-human-repr            = "1.0"
-hyper                 = { version = "0.14", features = ["client", "stream", "http1"] }
-hyper-rustls          = "0.23"
-jsonrpc-v2            = { version = "0.11", default-features = false, features = ["easy-errors", "macros", "bytes-v05"] }
-lazy_static           = "1.4"
-libipld               = { version = "0.14", default-features = false, features = ["dag-cbor", "dag-json", "derive"] }
-libipld-core          = "0.14"
-libipld-macro         = "0.14"
-libp2p                = { version = "0.49", default-features = false }
-libp2p-bitswap        = { git = "https://github.com/hanabi1224/libp2p-bitswap", branch = "forest", features = ["compat"] }
-libsecp256k1          = "0.7"
-log                   = "0.4"
-lru                   = "0.8"
-multibase             = "0.9"
-multihash             = "0.16"
-nonempty              = "0.8.0"
-num-bigint            = "0.4"
-num-derive            = "0.3"
-num-rational          = "0.4"
-num-traits            = "0.2"
-num_cpus              = "1.14"
-once_cell             = "1.15"
-pin-project-lite      = "0.2"
-pretty_env_logger     = "0.4"
-prometheus            = "0.13"
-prometheus-client     = "0.18"
-quickcheck            = "1"
-quickcheck_macros     = "1"
-rand                  = "0.8"
-rayon                 = "1.5"
-regex                 = "1.6"
-rpassword             = "7.0"
-serde                 = "1.0"
-serde_ipld_dagcbor    = "0.1"
-serde_json            = "1.0"
-serde_with            = { version = "2.0.1", features = ["chrono_0_4"] }
-sha2                  = "0.10.5"
-smallvec              = "1.9"
-structopt             = "0.3"
-tempfile              = "3.3"
-thiserror             = "1.0"
-time                  = "0.3"
-tokio                 = "1.21"
-tokio-stream          = "0.1"
-tokio-util            = "0.7.0"
-toml                  = "0.5"
-tower-http            = "0.3"
-url                   = { version = "2.3", features = ["serde"] }
-which                 = "4.3"
+filecoin-proofs-api = "12.0"
+flume = "0.10"
+fs_extra = "1.2"
+futures = "0.3"
+futures-util = "0.3"
+fvm = "2.1"
+fvm_ipld_amt = "0.4"
+fvm_ipld_bitfield = "0.5"
+fvm_ipld_blockstore = "0.1"
+fvm_ipld_car = "0.5"
+fvm_ipld_encoding = "0.2"
+fvm_ipld_hamt = "0.5"
+fvm_shared = "2.0"
+git-version = "0.3"
+hex = "0.4"
+http = "0.2.8"
+human-repr = "1.0"
+hyper = { version = "0.14", features = ["client", "stream", "http1"] }
+hyper-rustls = "0.23"
+jsonrpc-v2 = { version = "0.11", default-features = false, features = ["easy-errors", "macros", "bytes-v05"] }
+lazy_static = "1.4"
+libipld = { version = "0.14", default-features = false, features = ["dag-cbor", "dag-json", "derive"] }
+libipld-core = "0.14"
+libipld-macro = "0.14"
+libp2p = { version = "0.50", default-features = false }
+libp2p-bitswap = { git = "https://github.com/hanabi1224/libp2p-bitswap", branch = "forest-libp2p@0.50", features = [
```

Comment:
The taplo format change might come from this line?

Reply:
Have you tried not using `@` in the branch name? Perhaps it kills taplo for some bizarre reason?

Reply:
Nah, I just checked, it's related to overall line length. Would be nice if you'd shorten the name.

Reply:
Alternatively, as workaround, you can try increasing this https://github.com/ChainSafe/forest/blob/main/taplo.toml#L3. It might be worth filing a bug as well to the taplo project.

Reply:
Thanks, it's indeed the line width. Fixed by doing 
```
libp2p-bitswap.branch   = "forest-libp2p@0.50"
libp2p-bitswap.features = ["compat"]
libp2p-bitswap.git      = "https://github.com/hanabi1224/libp2p-bitswap"
```